content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
+
+By following this guide, you will be able to:
+
+- Set up Kubeflow in an AKS Cluster
+- Create and compile a pipeline that can:
+	- Preprocess data
+	- Train a model
+	- Register the model to ACR ([Azure Container Registry](https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/acr-template?view=azure-devops))
+	- Profile the model to optimize compute resources in AML (Azure Machine Learning)
+	- Deploy the model to AML
+- Interact with and customize your deployment
+- Test and use your deployed model
+
+When your pipeline is finished, you should be able to see a registered image, model, and deployment in your Azure ML workspace. You will then be able to visit the scoring URI and upload images for scoring in real time.
+
+## Set up your environment
+### Download the project files
+This tutorial uses the Azure Pipelines example in the Kubeflow examples repo. You can optionally use a pipeline of your own, but several key steps may differ.
+
+Clone the project files and go to the directory containing the [Azure Pipelines (Tacos and Burritos)](https://github.com/kubeflow/examples/tree/master/pipelines) example:
+```
+git clone https://github.com/kubeflow/examples.git
+cd examples/pipelines/azurepipeline
+```
+As an alternative to cloning, you can download the [Kubeflow examples repository zip file](https://github.com/kubeflow/examples/archive/master.zip).
+## Deploy Kubeflow
+If you don't already have one, create an Azure account. If you have not used Azure services before, you can recieve up to [1 year of free services and free credits.](https://azure.microsoft.com/en-ca/free/)
+
+> Note: that some of the services used in this guide may not be included in the free services, but can be covered by free credits. 
+
+First, install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest), then follow the instructions in the [guide to deploying Kubeflow on Azure](https://www.kubeflow.org/docs/azure/deploy/install-kubeflow/). 
+> Ensure that the agent size you use has the proper memory and storage requirements. For the Azure Pipelines example, **56 GiB** of memory are needed and **premium storage** must be available. Use [this guide](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes) to choose the right agent size for your deployment. (We chose an agent size of Standard_DS13_v2.)
+## Configuring Azure Resources
+### Create an ML Workspace in Azure
+An ML Workspace in Azure is where all of your ML models will be stored and updated. There is also support for managing your active deployments, which will be displayed later in this tutorial.
+
+Go to portal.azure.com and click on your resource group. From there, select the “add a new resource” option. Search for ‘Machine Learning Studio Workspace’ and use the default options, taking note of the name you decide for it. 
```

Comment:
Step-by-step instructions like this are much easier to follow if you format them as a numbered list.

Please format this section and others like it as a numbered list, with an introductory sentence for each list explaining what people achieve by following the steps. That makes it much easier to remember where you are in a doc and know what you've done, as you flip between the doc and your terminal/UI.

Reply:
Also, format all quotes from UI text in bold, so people can find it easily. For example:

```
1. Go to `portal.azure.com` and click on your resource group.
1. Select the **add a new resource** option.
```

The above renders like this:

1. Go to `portal.azure.com` and click on your resource group.
1. Select the **add a new resource** option.